flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/AsyncSinkBaseBuilder.java

```diff
@@ -40,8 +40,9 @@
     private Integer maxBatchSize;
     private Integer maxInFlightRequests;
     private Integer maxBufferedRequests;
-    private Long flushOnBufferSizeInBytes;
+    private Long maxBatchSizeInBytes;
     private Long maxTimeInBufferMS;
+    private Long maxRecordSizeInBytes;
```

Comment:
Why are these not primitive? Are they nullable?

Reply:
Yeah, they're nullable, so we can pull in defaults if null.